storybook/stories/Examples/LineChart.stories.tsx

```diff
@@ -550,7 +550,9 @@ export const HighlightAndZoom = {
             width={800}
             height={400}
             data={data}
+            // @ts-expect-error we're changing CategoricalChartState and activeLabel is no longer going to be there - what do we do with this story?
             onMouseDown={e => setZoomGraph(prev => ({ ...prev, refAreaLeft: e.activeLabel }))}
```

Comment:
We'll have to give a way to make this possible. It doesn't have to happen here necessarily, it could happen in Line or elsewhere